ui/ui-expenses/src/components/header/header.jsx

```diff
@@ -7,9 +7,7 @@ const Header = ({ name, avatar, title }) => (
   <div className={styles.header}>
     <img src={avatar} alt={name} className={styles.avatar} />
     <h1 className={styles.title}>{title}</h1>
-    {
-      title === 'envoi' && <Send />
-    }
+    <Send />
```

Comment:
The purpose of this button is to send recorded expenses to the acocuntant.
To add an expense to the list, there will be a button at the bottom of the page.
That's why there was a condition on this component

Reply:
**acocuntant**

Reply:
it was late !!!
*accountant*

Reply:
i'm ok with you @delphinemillet !
I let you do the design ?
But we can actually think of a generic button component with just send event ?

Reply:
Here I'm not sure we need a generic component: design, position in the screen and purpose of the action totally different for the 2 buttons.
ok for the design, I take the point !

Reply:
@delphinemillet the generic button could be used into expenses AND invoices.
If you are ok, maybe we should do a `packages` with all our common components ?
 - Maybe `packages/ui-components` ?

Reply:
new issues for this point : #26

Reply:
ok, I didn't understood what you meant 😵 
So have generic components in a common package is a good idea 👍

Reply:
@guillaumecrespel do you agree to put back the condition on the display of `Send` ? (initial way or another if you prefer)